src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -844,6 +882,12 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
             const auto altEnabled = WI_IsFlagSet(modifiers, static_cast<uint32_t>(VirtualKeyModifiers::Menu));
             const auto shiftEnabled = WI_IsFlagSet(modifiers, static_cast<uint32_t>(VirtualKeyModifiers::Shift));
 
+            if (_TrySendMouseEvent(point, true))

```

Comment:
If the user is holding down <kbd>shift</kbd>, suppress mouse events for now. We might need a setting for this.

Reply:
This will be true of all three handlers.

Reply:
maybe the best place to put that check is in `_TrySend`

Reply:
I'll create a follow up task creating a setting.

Reply:
ty